shopfloor_reception_packaging_dimension/tests/test_set_package_dimension.py

```diff
@@ -0,0 +1,161 @@
+        )
+
+    def test_scan_lot_ask_for_dimension(self):
+        self.product_a.tracking = "none"
+        selected_move_line = self.picking.move_line_ids.filtered(
+            lambda l: l.product_id == self.product_a
+        )
+        self.assertTrue(self.product_a.packaging_ids)
+        response = self.service.dispatch(
+            "set_lot_confirm_action",
+            params={
+                "picking_id": self.picking.id,
+                "selected_line_id": selected_move_line.id,
+            },
+        )
+        self.data.picking(self.picking)
+        selected_move_line = self.picking.move_line_ids.filtered(
+            lambda l: l.product_id == self.product_a
+        )
+        self._assert_response_set_dimension(
+            response, self.picking, selected_move_line, self.product_a_packaging
+        )
+
+    def test_set_packaging_dimension(self):
+        selected_move_line = self.picking.move_line_ids.filtered(
+            lambda l: l.product_id == self.product_a
+        )
+        self.service.dispatch(
+            "set_packaging_dimension",
+            params={
+                "picking_id": self.picking.id,
+                "selected_line_id": selected_move_line.id,
+                "packaging_id": self.product_a_packaging.id,
+                "height": 55,
+                "qty": 34,
+                "barcode": "barcode",
+            },
+        )
+        self.assertEqual(self.product_a_packaging.height, 55)
+        self.assertEqual(self.product_a_packaging.barcode, "barcode")
+        self.assertEqual(self.product_a_packaging.qty, 34)
+
+    def test_set_multiple_packaging_dimension(self):
+        line = self.picking.move_line_ids.filtered(
+            lambda l: l.product_id == self.product_c
+        )
+        # Set the weight but other dimension are required
+        self.product_c_packaging_2.max_weight = 200
+        response = self.service.dispatch(
+            "set_packaging_dimension",
+            params={
+                "picking_id": self.picking.id,
+                "selected_line_id": line.id,
+                "packaging_id": self.product_c_packaging.id,
+                "height": 55,
+                "length": 233,
+            },
+        )
+        self.assertEqual(self.product_c_packaging.height, 55)
+        self.assertEqual(self.product_c_packaging.packaging_length, 233)
+        self._assert_response_set_dimension(
+            response,
+            self.picking,
+            line,
+            self.product_c_packaging_2,
+            message=self.msg_store.packaging_dimension_updated(
+                self.product_c_packaging
+            ),
+        )
+        response = self.service.dispatch(
+            "set_packaging_dimension",
+            params={
+                "picking_id": self.picking.id,
+                "selected_line_id": line.id,
+                "packaging_id": self.product_c_packaging_2.id,
+                "height": 200,
+                "weight": 1000,
```

Comment:
shopfloor_reception_packaging_dimension_mobile is never sending weight.
it sends max_weight
```suggestion
                "max_weight": 1000,
```

Reply:
Fixed